### PR TITLE
Updated contribution section.

### PR DESCRIPTION
**Pull request by AbhiVikrant**
* Updated and added pull request process.
* Updated and added code of conduct part.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -14,3 +14,65 @@
...
+
+###### **Enforcement**
+
+> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [INSERT EMAIL ADDRESS]. All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+>
+> Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+###### **Attribution**
+
+> This Code of Conduct is adapted from the [Contributor Covenant](http://contributor-covenant.org/), version 1.4, available at [http://contributor-covenant.org/version/1/4](http://contributor-covenant.org/version/1/4/).
```
Comment (praveenscience) [inline Readme.md]: None of this is what we use... Can you please update with how we generally contribute please?

**Review by praveenscience — CHANGES_REQUESTED** [review]
This is not we need. Please add how to contribute and what we learnt in the classes.

File: Readme.md (hunk 1 above)
Comment (AbhiVikrant) [reply Readme.md]: I got your points and changed it.

**Comment by praveenscience** [comment]
Closing due to conflicts.

**Pull request closed without merging**